x-pack/plugins/osquery/cypress/tasks/live_query.ts

```diff
@@ -49,7 +49,12 @@ export const checkResults = () => {
 export const typeInECSFieldInput = (text: string, index = 0) =>
   cy.getBySel('ECS-field-input').eq(index).type(text);
 export const typeInOsqueryFieldInput = (text: string, index = 0) =>
-  cy.react('OsqueryColumnFieldComponent').eq(index).react('ResultComboBox').type(text);
+  cy
+    .react('OsqueryColumnFieldComponent')
+    .eq(index)
+    .within(() => {
+      cy.getBySel('comboBoxInput').type(text);
+    });
```

Comment:
```suggestion
    .findBySel('comboBoxInput').type(text);
```

Reply:
findBySel is not chainable. getBySel doesnt work either. I believe we should leave it as it is - in within(). Wdyt?

Reply:
`findBySel` doesn't exist in `osquery`, could you add it and use it here?

Reply:
added that and it still didn't work as expected :( so I reverted the change.